README.md

```diff
@@ -196,7 +196,7 @@ module.exports = {
 };
 ```
 
-#### String
+#### string
```

Comment:
Let's always use "\`" for wrapping, i.e. "\`string\`", like we do for other headings

Reply:
Fixed this and removed table too